domains/brandon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "description": "Brandons's personal developer website",
-  "repo": "https://github.com/b-hexsoul",
+  "description": "Brandon Hexsel - Full Stack Engineer",
+  "repo": "https://github.com/b-hexsoul/b-hexsoul.github.io",
```

Comment:
You don't have the repo created. If you are having any trouble with github pages you can check out [their docs](https://guides.github.com/features/pages/)